Packs/ServiceNow/Integrations/ServiceNowv2/ServiceNowv2.py

```diff
@@ -575,30 +576,33 @@ def get_template(self, template_name: str) -> dict:
 
         return dic_template
 
-    def get_ticket_attachments(self, ticket_id: str) -> dict:
+    def get_ticket_attachments(self, ticket_id: str, sys_created_on=None) -> dict:
         """Get ticket attachments by sending a GET request.
 
         Args:
             ticket_id: ticket id
+            sys_created_on: string, when the attachment was created
 
         Returns:
             Response from API.
         """
-        return self.send_request('attachment', 'GET', params={'sysparm_query': f'table_sys_id={ticket_id}'})
+        return self.send_request('attachment', 'GET', params={
+            'sysparm_query': f'table_sys_id={ticket_id}^sys_created_on>{sys_created_on}'})
```

Comment:
did we verify this work if `sys_created_on` is not passed and the value here is `None` ?

Reply:
yeah, I run another commands that uses this. All looks good